apps/admin/frontend/src/components/app_routes.tsx

```diff
@@ -145,8 +144,8 @@ export function ElectionManager(): JSX.Element {
   // Election manager UI
   return (
     <Switch>
-      <Route exact path={routerPaths.ballotsList}>
-        <BallotListScreen />
```

Comment:
Is this a file that could be deleted now?

Reply:
The file containing `BallotsListScreen`

Reply:
Yup, meant to do that, thanks